src/AWSMetadataUtilities.jl

```diff
@@ -252,12 +258,12 @@ Get the required and optional parameters for a given operation.
 
 # Arguments
 - `input::String`: The input object for the given operation
-- `shapes::Dict{String, Any}`: Dictionary of all shapes for this AWS Service
+- `shapes::AbstractDict{String, Any}`: Dictionary of all shapes for this AWS Service
 
 # Returns
 - `Tuple(Dict, Dict)`: (required_parameters, optional_parameters)
 """
-function _get_function_parameters(input::String, shapes::Dict)
+function _get_function_parameters(input::String, shapes::AbstractDict)
```

Comment:
```suggestion
function _get_function_parameters(input::String, shapes::AbstractDict{String})
```
doesn't hurt to match the docstring